roles/openshift-prep/tasks/prerequisites.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: "Cleaning yum repositories"
+  command: "yum clean all"
+  
+- name: "Install required packages"
+  yum:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - wget
+    - git
+    - net-tools
+    - bind-utils
+    - bridge-utils
+    - bash-completion
+    - atomic-openshift-utils
```

Comment:
this doesn't work for centos7 w/o rhsm repos registered. Could you please ensure centos7 compatibility?